easydock/run_dock.py

```diff
@@ -209,10 +209,16 @@ def main():
                 del args_dict[arg]
             args.__dict__.update(args_dict)
 
-        start_init = time.time()
-        init_db(args.output, args.input, args.ncpu, args.max_stereoisomers, args.prefix)
-        end_init = time.time()
-        print(f'initializing database done in {(end_init - start_init):.2f}s', flush=True)
+        has_finished_init_protonation = check_init_status_db(args.output, ['smi_protonated', 'source_mol_block_protonated']) and args.protonation
+        has_finished_init_docking = check_init_status_db(args.output, ['dock_time'])
+        if  has_finished_init_protonation or has_finished_init_docking:
```

Comment:
I suggest to rename variables by removal `init` part from their names. This seems unnecessary.